locales/lt.lua

```diff
@@ -0,0 +1,281 @@
+  ['component_clip_extended']     = 'Prailginta apkaba',
+  ['component_clip_drum']         = 'Vidutine apkaba',
+  ['component_clip_box']          = 'Geriausia apkaba',
+
+  -- Flashlight
+  ['component_flashlight']        = 'Zibintuvelis',
+  
+  -- Scopes
+  ['component_scope']             = 'Optika',
+  ['component_scope_small']       = 'Maza optika',
+  ['component_scope_macro']       = 'Macro Optika',
+  ['component_scope_medium']      = 'Vidutine Optika',
+  ['component_scope_mounted']     = 'Primontuota Optika',
+  ['component_scope_advanced']    = 'Pazengusi Optika',
+  ['component_scope_zoom']        = 'Prailgina Optika',
+  ['component_scope_large']       = 'Didele Optika',
+  ['component_scope_nightvison']  = 'Naktinio matymo Optika',
+  ['component_scope_thermal']     = 'Silumine Optika',
+
+  -- Barrels / Suppressors
+  ['component_barrel']            = 'Vamzdis',
+  ['component_barrel_heavy']      = 'Sunkusis vamzdis',
+  ['component_suppressor']        = 'Supresorius',
+  ['component_compensator']       = 'Compensatorius',
+  
+  -- Grips
+  ['component_grip']              = 'Rankena',
+
+  -- Muzzles
+  ['component_muzzle_flat']       = 'Flat Muzzle Brake',
+  ['component_muzzle_tatical']    = 'Tactical Muzzle Brake',
+  ['component_muzzle_fat']        = 'Fat-End Muzzle Brake',
+  ['component_muzzle_precision']  = 'Precision Muzzle Brake',
+  ['component_muzzle_heavy']      = 'Heavy Duty Muzzle Brake',
+  ['component_muzzle_slanted']    = 'Slanted Muzzle Brake',
+  ['component_muzzle_split']      = 'Split-End Muzzle Brake',
+  ['component_muzzle_squared']    = 'Square Muzzle Brake',
+  ['component_muzzle_bellend']    = 'Bell-End Muzzle Brake',
+
+  -- Weapon Skins
+  ['component_skin_camo']         = 'Skaitmeninis Camo',
+  ['component_skin_brushstroke']  = 'Teptuko smugio Camo',
+  ['component_skin_woodland']     = 'Miskinis Camo',
+  ['component_skin_skull']        = 'Kaukuole',
+  ['component_skin_sessanta']     = 'Sesiasdesimt devyni',
+  ['component_skin_perseus']      = 'Perseus',
+  ['component_skin_leopard']      = 'Leopardas',
+  ['component_skin_zebra']        = 'Zebras',
+  ['component_skin_geometric']    = 'Geometriskas',
+  ['component_skin_boom']         = 'Boom!',
+  ['component_skin_patriotic']    = 'Patriotiskas',
+  ['component_luxary_finish']     = 'Prabanga',
+
+  -- Weapon Ammo
+  ['ammo_rounds'] = 'Kulka(os)',
+  ['ammo_shells'] = 'shell(s)',
+  ['ammo_charge'] = 'charge',
+  ['ammo_petrol'] = 'Litrai degalu',
+  ['ammo_firework'] = 'firework(s)',
+  ['ammo_rockets'] = 'rocket(s)',
+  ['ammo_grenadelauncher'] = 'grenade(s)',
+  ['ammo_grenade'] = 'grenade(s)',
+  ['ammo_stickybomb'] = 'bomb(s)',
+  ['ammo_pipebomb'] = 'bomb(s)',
+  ['ammo_smokebomb'] = 'bomb(s)',
+  ['ammo_molotov'] = 'cocktail(s)',
+  ['ammo_proxmine'] = 'mine(s)',
+  ['ammo_bzgas'] = 'can(s)',
+  ['ammo_ball'] = 'ball(s)',
+  ['ammo_snowball'] = 'snowball(s)',
+  ['ammo_flare'] = 'flare(s)',
+  ['ammo_flaregun'] = 'flare(s)',

```

Comment:
It's not translated. Also missing Lithuanian symbols like ąčęėšįųšžĄČĘĮŲŠŲŽ

Reply:
Yes, it’s missing those characters because it would display as squares. I will finish the translation a little bit later.

It’s kinda weird tho, some characters show up normally, but some as squares.